integration/build.gradle

```diff
@@ -1 +1,36 @@
-// TODO! Maybe get started by copy/pasting from data-catalog?
+import org.springframework.boot.gradle.plugin.SpringBootPlugin
+
+plugins {
+    id 'io.spring.dependency-management'
+    id 'bio.terra.test-runner-plugin'
+}
+
+dependencyManagement {
+    imports {
+        mavenBom(SpringBootPlugin.BOM_COORDINATES)
+    }
+}
+
+// In `terra-data-catalog` the only `repositories { }` stanzas are in
+// the top-level `buildSrc` directory. But for some reason it does not get
+// inherited (or whatever the mechanism is) in this project.
```

Comment:
This is because this subproject doesn't have a dependency on any of your plugin projects (which is what `buildSrc` is defining). Change `plugins` to be
```
plugins {
    id 'java-application-conventions'
    id 'io.spring.dependency-management'
    id 'bio.terra.test-runner-plugin'
 }
```
and you can delete this `repositories` block.

Reply:
Thanks... I knew I was off on something... not sure why I originally deleted `java-application-conventions`